Assets/Scripts/Player/PlayerController.cs

```diff
@@ -91,4 +105,50 @@ public class PlayerController : MonoBehaviour {
             transform.rotation = Quaternion.Euler(0f, angle, 0f);
         }
     }
+    
+    private void ExecuteLadderMove() {
+        if (Input.GetButtonDown("Jump")) {
+            DisengageLadder();
+            verticalSpeed = Vector3.up * jumpHeight;
+            controller.Move(verticalSpeed * Time.deltaTime);
+            return;
+        }
+        
+        float verticalClimb = 0.0f;
+        
+        if (activeLadder.verticalClimbDirection == Ladder.VerticalClimbDirection.Up) {
```

Comment:
Could add modifier (1 or -1) to enum and access that directly to multiply with the axis, but this way is good as well